src/http/server.cr

```diff
@@ -138,8 +138,8 @@ class HTTP::Server
     end
   end
 
-  def bind
-    @server ||= TCPServer.new(@host, @port)
+  def bind(reuse_port = false)
```

Comment:
Allright. Do we need the property is we add the argument to both `listen` and `bind`? Just say "yes" or "no" :-)